bot.ml

```diff
@@ -732,7 +732,7 @@ let callback _conn req body =
             )
           else if
             string_match ~regexp:(f "@%s: [Rr]un CI now" bot_name) body
-            && Option.is_some comment_info.pull_request
+            && Bool.equal comment_info.issue.pull_request true
```

Comment:
The `pull_request` field from the `comment_info` record doesn't hold a value when the comment originates from a PR, so we rely on the `pull_request` boolean from the `issue` field instead (this was how it was handled previously).

Reply:
```suggestion
            && comment_info.issue.pull_request
```